k8s/voucher/manifest/manifests.yaml.template

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: $name-server
+spec:
+  selector:
+    matchLabels:
+      run: $name-app
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: $name-app
+    spec:
+      containers:
+      - name: voucher-server
+        image: $voucherImage
+        imagePullPolicy: Always
+        args:
+          - -project=$projectId
+          - -auth=$enableServerAuth
+          - -username=$username
+          - -passhash=$passHash
```

Comment:
One recommendation here, if container could get this variable from ENV vars:

- Create a secret to store this value. 
- Import from secret to a `env` in the container

Reply:
I see, is the goal here to hide the strings from the YAML?
The passhash is already a hashed value (using bcrypt) to prevent leakage concerns.